### DISCUSSION TITLE
Cannot install the latest version about CGAL.

### DISCUSSION THREAD

**Discussion by houwpx** [CGAL/cgal#6633] [Q&A] [answered]
Why I run the this command 'sudo apt-get install libcgal-dev' on ubuntu. It doesn't install the latest version. Is there a way to force install 5.4 version?

Please see the following installition informations

root@4fa115e753d8:/# sudo apt-get install libcgal-dev
Reading package lists... Done
Building dependency tree       
Reading state information... Done
libcgal-dev is already the newest version (5.0.2-3).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
root@4fa115e753d8:/# date
Wed Jun  1 02:03:15 CST 2022
root@4fa115e753d8:/# 

-Ryan

**Comment by sloriot** [comment]
There is nothing specific to CGAL in your question. That's the behavior of your GNU/Linux distribution.

**Comment by MaelRL** [comment, marked as answer]
Each Ubuntu version has its own CGAL version, see https://packages.ubuntu.com/search?keywords=cgal.

To get the latest, either download from github, or you can also have a look at https://www.cgal.org/FAQ.html#debian_packages.
